### PR TITLE
Update v5.3-performance-benchmarking-with-tpch.md

### PR DESCRIPTION
**Pull request by heipark**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@heipark Could you please resolve the conflict? Thanks~

**Comment by TomShawn** [comment]
@ilovesoup Could you please take a look? Thanks~

File: benchmark/v5.3-performance-benchmarking-with-tpch.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ title: TiDB TPC-H 100 性能对比测试报告 - v5.3 MPP 模式对比 Greenplum
 
 ## 测试概况
 
-本次测试对比了 TiDB v5.3 MPP 模式下和主流分析引擎例如 Greenplum 和 Apache Spark 最新版在 TPC-H 100 下的性能表现。结果显示，TiDB v5.3 MPP 模式下相对这些方案有 2-3 倍的性能提升。
+本次测试对比了 TiDB v5.3 MPP 模式下和主流分析引擎例如 Greenplum 和 Apache Spark 最新版在 TPC-H 100GB 数据下的性能表现。结果显示，TiDB v5.3 MPP 模式下相对这些方案有 2-3 倍的性能提升。
```
[the remark this answers is not in the document]
Comment (heipark) [reply benchmark/v5.3-performance-benchmarking-with-tpch.md:9]: 上面是100GB，这里是100 GB，可以统一为100GB ：）

**Review by en-jin19 — APPROVED** [review]
LGTM

**Review by zanmato1984 — APPROVED** [review]
LGTM

**Comment by en-jin19** [comment]
The changes in this PR is made in the new PR https://github.com/pingcap/docs-cn/pull/8601

**Pull request closed without merging**